File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_ckprstdn.java
// EXPECTED: 150
// Reference emit for CKPRSTDN — CKPRST COPY-linked status 88 RC sum.
public final class CkprstdnRef {
  static int statusRc(String status) {
    if ("I".equals(status)) return 10;
    if ("A".equals(status)) return 20;
    if ("C".equals(status)) return 30;
    if ("F".equals(status)) return 40;
    if ("R".equals(status)) return 50;
    return 99;
  }

  public static void main(String[] args) {
    int total = 0;
    for (String s : new String[] {"I", "A", "C", "F", "R"}) {
      total += statusRc(s);
    }
    System.out.println(total);
  }
}

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_ckprstdn.cs
// EXPECTED: 150
// Reference emit for CKPRSTDN — CKPRST COPY-linked status 88 RC sum.
using System;
public static class CkprstdnRef {
  static int StatusRc(string status) {
    if (status == "I") return 10;
    if (status == "A") return 20;
    if (status == "C") return 30;
    if (status == "F") return 40;
    if (status == "R") return 50;
    return 99;
  }

  public static void Main() {
    int total = 0;
    foreach (var s in new[] { "I", "A", "C", "F", "R" }) {
      total += StatusRc(s);
    }
    Console.WriteLine(total);
  }
}
